shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/resources/BridgeExecutor.java

```diff
@@ -155,6 +164,12 @@ public BridgeExecutor build() {
             bridgeExecutorSpec.setCustomerId(customerId);
             bridgeExecutorSpec.setProcessorName(processorName);
 
+            if (processorType != null) {
```

Comment:
@kostola Why this `!= null` check only on `processorType`? All the other fields are required e.g `customerId` so I think we should be consistent in our approach here.

Reply:
Done :+1: